frontend/package.json

```diff
@@ -79,7 +79,7 @@
   "scripts": {
     "clean": "rm -rf dist/ && rm -rf .cache/",
     "serve": "webpack serve --config webpack.dev.js",
-    "build": "webpack --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
```

Comment:
How/where is `frontend/package.json` used? Why do we use different configs for `build` and `serve`?

Reply:
build just assemble bundles, in production mode bundles are minimises. for serve it's just dev only configuration which runs dev server and proxy, so dev config doesn't affect production and it may contain anything. ideally production and dev config should contain essential parts common, so this part is in `webpack.common.js`